data/vibe/data/json.d

```diff
@@ -1868,6 +1868,22 @@ struct JsonSerializer {
 				case Json.Type.float_: return cast(T)m_current.get!double;
 				case Json.Type.bigInt: return cast(T)m_current.bigIntToLong();
 			}
+		} else static if (is(T == enum)) {
+			const T value = () {
```

Comment:
Since `static if` doesn't create a scope, this could be shortened to just
```
			static if (hasPolicyAttributeL!(ByNameAttribute, Traits.Policy, Traits.Attributes)) {
				const T value = readValue!(Traits, string)().to!T();
			} else {
				const T value = cast(T)readValue!(Traits, OriginalType!T)();
			}
```

Maybe I need to look closer, but shoudn't the `to!T` throw a `ConvException` in the `string` case, circumventing the `JSONException` in the bottom?

Reply:
You're right, it should throw. I wonder if it is used now =) It should generate a 500 response I suppose.